yarn.lock

```diff
@@ -11688,6 +11688,11 @@ graphql-tag@^2.11.0:
   dependencies:
     tslib "^2.1.0"
 
+graphql-ws@5.10.1:
+  version "5.10.1"
+  resolved "https://registry.yarnpkg.com/graphql-ws/-/graphql-ws-5.10.1.tgz#c5402a62f1be13e244ab90a6306b94f643577312"
+  integrity sha512-MKm/3SRd1vj5Km8NaujsgeGRTKZQjUN5HRnIMJ8dL2UznKoxvrtQyJqTmqJt0f6vQd9AooDg/+baXo3huiY4Ew==
+
```

Comment:
we now have multiple instances of graphql-ws installed, can you bump all graphql-ws deps to the same version?

Reply:
I only see it installed for example  
![image](https://user-images.githubusercontent.com/44710980/187229750-120fb128-db59-416b-8f1a-cde8098ecd2b.png)